js/src/setup-mc/setup-stepper/setup-paid-ads/paid-ads-setup-sections.js

```diff
@@ -76,6 +78,13 @@ export default function PaidAdsSetupSections( {
 	const isBillingCompleted =
 		billingStatus?.status === GOOGLE_ADS_BILLING_STATUS.APPROVED;
 
+	const { data: budgetData, loading } =
+		useFetchBudgetRecommendationEffect( countryCodes );
+	const { country = '', daily_budget: dailyBudget } = getHighestBudget(
+		budgetData?.recommendations || []
+	);
+	const multipleRecommendations = budgetData?.recommendations.length > 1;
```

Comment:
@ankitrox The code is duplicated here. Can we move it to a hook which can be reused elsewhere?

Reply:
Outdated with latest changes.